cdap-docs/build.sh

```diff
@@ -171,18 +174,23 @@ function build_docs_outer_level() {
 ################################################## current
 
 function build_all() {
+  echo "========================================================================="
   echo "Building GitHub Docs."
-  ./build.sh docs-github $ARG_2 $ARG_3
+  echo "========================================================================="
+  run_command docs-github-part $ARG_2 $ARG_3
```

Comment:
Here's what I would see needs to be tested, either manually or automatically: 
Outer level: 
```
build.sh docs
build.sh docs-web
build.sh docs-github
build.sh docs all
```
Check that the correct versions of Google Analytics are set in the output.
Check that builds can be killed: cntrl-c.

Inner Level (in each manual or sub-directory):
```
build.sh docs
```
If all these work, we should be good.